hardware/Limitless.cpp

```diff
@@ -238,15 +238,17 @@ bool CLimitLess::StartHardware()
 				{
 					if (!AddSwitchIfNotExits(3, "AppLamp Group3"))
 					{
-						AddSwitchIfNotExits(4, "AppLamp Group4");
+						if (!AddSwitchIfNotExits(4, "AppLamp Group4"))
+						{
+							if (m_BridgeType == LBTYPE_V6)
```

Comment:
This was removed for a reason.
If you switch from bridge types and do not change the light type, the 5 lamp groups still exist, but the bridge light never gets created
Leaving in where it was before does not do any harm